client/app/configuration/configuration.controller.js

```diff
@@ -31,6 +35,18 @@ angular.module('App').controller(
       this.selectTypeOfGuide('domainHosting');
     }
 
+    buildSummitData() {
```

Comment:
I think you should rebase (code removed in https://github.com/ovh-ux/ovh-manager-web/commit/7e0c43aa5fc814a61f3cea8c325713b5cc8237ac)

Reply:
Thanks for the tip. Have rebased